src/components/VerificationOverlay.js

```diff
@@ -0,0 +1,34 @@
+const VerificationOverlay = () => {
+  return (
+    <div className="absolute p-2">
+      <div className="space-y-6 w-full bg-neutral-100 p-9 rounded-2xl ">
+        <div className="text-blue-600 dark:text-white text-3xl font-extrabold">
+          Account Verification
+        </div>
+        <p>
+          A verification link has been sent to{" "}
+          <span className="text-blue-600">augustine@example.com.</span> Please
+          check your inbox and click on the link to complete your registration.
+        </p>
+        <div>
+          <a href="#" className="text-blue-600 underline  font-bold">
+            Change Email
+          </a>
+        </div>
+        <div className="flex flex-col space-y-3">
+          <button className="bg-blue-600 hover:bg-blue-600 text-white px-6 py-3 rounded-md">
+            Open mail app
+          </button>
+        </div>
+        <p className="text-neutral-500">
+          Did not receive the mail? Check your spam
+          <span className="absolute  right-12">
```

Comment:
Remove line 25 to 27

Reply:
That will be removing the resend mail timer

Reply:
Hi @Zer0SZN please if you’re still having issues implementing these changes, kindly reach me.